virttest/qemu_monitor.py

```diff
@@ -2302,6 +2302,20 @@ def block_resize(self, device, size):
         cmd = "block_resize device=%s,size=%s" % (device, size)
         return self.send_args_cmd(cmd)
 
+    def get_block_virtual_size(self, dev_dict={}):
+        """
+        Get block virtual-size of specified block device
+        from monitor's info block command.
+
+        :param dev_dict: define device dict from block info.
+        :return: Matched block virual size.
+        """
+        blocks_info = self.info("block")
+        for dict in blocks_info:
+            if dev_dict['device'] in dict['device']:
```

Comment:
dev_dict is a default param, handle KeyError

Reply:
has updated, it's no need to set the default value for this param,thanks~